doc/contributing.md

```diff
@@ -579,7 +579,7 @@ returning the `pygmt.Figure` object:
 def test_my_plotting_case():
     "Test that my plotting method works"
     fig = Figure()
-    fig.basemap(region=[0, 360, -90, 90], projection='W7i', frame=True)
+    fig.basemap(region=[0, 360, -90, 90], projection='W15c', frame=True)
```

Comment:
```suggestion
    fig.basemap(region=[0, 360, -90, 90], projection="W15c", frame=True)
```

Reply:
Thanks @seisman! Good spotting! Saw it by myself while working on PR #2018 :wink:. Done in https://github.com/GenericMappingTools/pygmt/pull/2107/commits/c1d3f05a42c48f9720f80acc1829341faa81ae8e.